pages/index.tsx

```diff
@@ -10,6 +14,137 @@ const Home = () => {
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Create your cookie cutter project templates with Starters.
+              </a>
+            </>
+          }
+          title="Project Template"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="left"
+          imageSrc={placeholder}
+          subtitle={
+            <>
+              Test-driven development using{' '}
+              <a
+                href="https://github.com/pytest-dev/pytest"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                pytest
+              </a>
+              , produce well-documented code using{' '}
+              <a
+                href="http://www.sphinx-doc.org/en/master/"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Sphinx
+              </a>
+              , create linted code with support for{' '}
+              <a
+                href="https://github.com/PyCQA/flake8"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                flake8
+              </a>
+              ,{' '}
+              <a
+                href="https://github.com/PyCQA/isort"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                isort
+              </a>{' '}
+              and{' '}
+              <a
+                href="https://github.com/psf/black"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                black
+              </a>{' '}
+              and make use of the standard Python logging library.
+            </>
+          }
+          title="Coding Standards"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="right"
+          imageSrc={placeholder}
+          subtitle="Apache Spark, Pandas, Dask, Matplotlib, Plotly, fsspec, Apache Airflow, Jupyter Notebook and Docker."
+          title="Integrations"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="left"
+          buttonLink="https://kedro.readthedocs.io/en/stable/05_data/01_data_catalog.html"
+          buttonText="Learn more"
+          imageSrc={placeholder}
+          subtitle="A series of lightweight data connectors used to save and load data across many different file formats and file systems, including local and network file systems, cloud object stores, and HDFS. The Data Catalog also includes data and model versioning for file-based systems."
+          title="Data Catalog"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="right"
+          buttonLink="https://kedro.readthedocs.io/en/stable/10_deployment/01_deployment_guide.html"
+          buttonText="Explore"
+          imageSrc={placeholder}
```

Comment:
While this section is not avaliable on the design on Figma, it does seem that the image is avaliable on the box note - we can also slot this in in this PR
<img width="874" alt="image" src="https://user-images.githubusercontent.com/27775658/155740624-fd483f23-eab7-4b73-8daf-942e5f3c389e.png">